content/3.Contributing/1.Guidelines.md

```diff
@@ -0,0 +1,11 @@
+::alert{type="info"} ℹ️ | This documentation is for contributing to the [SLF](https://github.com/dswan36/SillyLittleFiles) projects on GitHub, and as such, is not relivant to personal setup ::
```

Comment:
Syntax for infobox is somehow borked up and is causing it to not render properly. Check & Fix